selfdrive/car/toyota/carstate.py

```diff
@@ -94,6 +97,8 @@ def __init__(self, CP):
     self.shifter_values = self.can_define.dv["GEAR_PACKET"]['GEAR']
     self.left_blinker_on = 0
     self.right_blinker_on = 0
+    self.offset = 0.
+    self.isoffset = 0
```

Comment:
this should be a bool called: `is_offset_set` I think

Reply:
And done!